arch/powerpc/include/asm/pci-bridge.h

```diff
@@ -212,19 +212,17 @@ struct pci_dn {
 	struct eeh_dev *edev;		/* eeh device */
 #endif
 #define IODA_INVALID_PE		0xFFFFFFFF
-#ifdef CONFIG_PPC_POWERNV
+#ifdef CONFIG_PCI_IOV
```

Comment:
This is changing the behavior for powernv, is that intended?  Previously, pe_number, vf_index, and mps were defined even if CONFIG_PCI_IOV was not defined.

Reply:
Probably dont need vf_index but pe_number is going to be needed. So i guess you can just move pe_number down into the CONFIG_PCI_IOV section that you deleted and moved up.

Reply:
Yes, from the powernv developers was that SR-IOV will only be enabled in PowerNV platforms. That is not the case anymore. Also, one cannot make the assumption that Linux Image will be loaded in Opal or Lpar. Code must live in both.

Reply:
I take back my earlier comment, we do need vf_index.

Reply:
Following Steve's comment I might need to reorganize the compiler flags a bit. Expect a change.